apps/molgenis-components/src/components/tables/TableSimple.vue

```diff
@@ -0,0 +1,210 @@
+</template>
+
+<script>
+/** Data table. Has also option to have row selection. Selection events must be handled outside this view. */
+export default {
+  props: {
+    /** the column names */
+    columns: Array,
+    /** list of rows matching column names */
+    rows: Array,
+    /** set to create select boxes that will yield this columns value when selected. */
+    selectColumn: String,
+    /** default value */
+    defaultValue: Array,
+  },
+  data: function () {
+    return {
+      selectedItems: [],
+    };
+  },
+  watch: {
+    selectedItems() {
+      this.$emit("input", this.selectedItems);
+    },
+  },
+  created() {
+    if (this.defaultValue instanceof Array) {
+      this.selectedItems = this.defaultValue;
+    } else {
+      this.selectedItems.push(this.defaultValue);
+    }
+  },
+  computed: {
+    tableHover() {
+      return this.selectColumn || (this.$listeners && this.$listeners.click);
+    },
+    hasColheader() {
+      return (
+        this.selectColumn ||
+        !!this.$slots["colheader"] ||
+        !!this.$scopedSlots["colheader"] ||
+        !!this.$slots["rowheader"] ||
+        !!this.$scopedSlots["rowheader"]
+      );
+    },
+  },
+  methods: {
+    flattenObject(object) {
+      let result = "";
+      if (typeof object === "object") {
+        Object.keys(object).forEach((key) => {
+          if (object[key] === null) {
+            //nothing
+          } else if (typeof object[key] === "object") {
+            result += this.flattenObject(object[key]);
+          } else {
+            result += " " + object[key];
+          }
+        });
+      } else {
+        result = object;
+      }
+      return result.replace(/^\./, "");
+    },
+    isSelected(row) {
+      return (
+        this.selectedItems != null &&
+        this.selectedItems.includes(row[this.selectColumn])
+      );
+    },
+    onRowClick(row) {
+      if (this.selectColumn) {
+        if (this.isSelected(row)) {
+          /** when a row is deselected */
+          this.selectedItems = this.selectedItems.filter(
+            (item) => item !== row[this.selectColumn]
+          );
+          this.$emit("deselect", row[this.selectColumn]);
+        } else {
+          /** when a row is selected */
+          this.selectedItems.push(row[this.selectColumn]);
+          this.$emit("select", row[this.selectColumn]);
+        }
+      } else {
+        this.$emit("click", row);
+      }
+    },
+  },
+};
+</script>
+
+<style scoped>
+th,
+td {
+  text-align: left;
+}
+</style>
+
+<docs>
+
+<template>
+<div>
+  <div>
+    <TableSimple
+        v-model="selectedItems1"
+        selectColumn="lastName"
+        :defaultValue="selectedItems1"
+        :columns="['firstName','lastName','tags']"
+        :rows="[{'firstName':'Donald','lastName':'Duck'},{'firstName':'Scrooge','lastName':'McDuck','tags':['blue','green']}]"
+        @select="select"
+        @deselect="deselect"
+    />
+    SelectedItems: {{ selectedItems1 }}
+  </div>
+
```

Comment:
Wrap each item inside of a DemoItem component? Do we want that everywhere for consistency?